Global/JetBrains.gitignore

```diff
@@ -1,6 +1,9 @@
 # Covers JetBrains IDEs: IntelliJ, RubyMine, PhpStorm, AppCode, PyCharm, CLion, Android Studio and WebStorm
 # Reference: https://intellij-support.jetbrains.com/hc/en-us/articles/206544839
 
+# JetBrains Compile-time Files
+.idea/
```

Comment:
This makes all the subsequent rules unnecessary. I'd prefer this template reflect what should be in version control - are you able to elaborate on what files are not covered by this current set of rules?

Reply:
I'm deferring to this part of the JetBrains documentation in the link at the top of the gitignore template:

> Here is what you need to share:
> - All the files under `.idea` directory in the project root except the `workspace.xml`, `usage.statistics.xml`, and `tasks.xml` files which store user specific settings

It's not clear to me what's in those files that might be per-user (it's been a while since I've used any JetBrains IDEs to see for myself). Care to elaborate on why you think those can be safely ignored?